packages/venia-ui/lib/queries/addSimpleProductsToCart.graphql

```diff
@@ -1,25 +0,0 @@
-mutation addSimpleProductToCart(
```

Comment:
I think this deletion makes this a `major`. Please update the label if you agree.

Reply:
Correct. Possible that folks have imported directly.